### PR TITLE
AWS support

### PR DESCRIPTION
**Pull request by artem-belov**
Amazon Web Services support via Amazon Elastic Container Service for Kubernetes

## Description
AWS EKS support for nsm uses.

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [x] Tested locally
- [ ] Have not tested

Past all tests except 
-  TestClusterInfo
- TestNSCDiesMultiNode

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: .mk/aws.mk [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2018 Cisco and/or its affiliates.
```
Comment (nickolaev) [inline .mk/aws.mk]: It's 2019

File: docs/aws-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Put key ID and secret into environment variables:
+* NSM_AWS_ACCESS_KEY_ID
+* NSM_AWS_SECRET_ACCESS_KEY
+
+# Prerequisites
+
+## Setup AWS.
+
+```shell
+* make aws-init
```
Comment (nickolaev) [inline docs/aws-quickstart.md]: please remove the `*` here.

File: scripts/aws/create-kubernetes-cluster.py [hunk 3]
```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python
```
Comment (nickolaev) [inline scripts/aws/create-kubernetes-cluster.py]: I would always prefer `go` if possible.
@edwarnicke @fkautz any opinions here?

File: docs/aws-quickstart.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+```
+
+## Use AWS environment.
+
+Before start using of AWS, please execute the following command:
+
+```shell
+export CLUSTER_RULES_PREFIX=gke
```
Comment (nickolaev) [inline docs/aws-quickstart.md]: Can we have `.env/aws.env` similar to [.env/gke.env](https://github.com/networkservicemesh/networkservicemesh/blob/master/.env/gke.env)?

File: scripts/aws/create-kubernetes-cluster.py (hunk 3 above)
Comment (artem-belov) [reply scripts/aws/create-kubernetes-cluster.py]: I discussed it with Ed and changed scripts to Go lang. 
I keep python scripts in repository if somebody will need it.

**Pull request merged**